packages/cadl-ranch-specs/http/models/inheritance/main.cadl

```diff
@@ -1,60 +1,112 @@
-import "@cadl-lang/rest";
 import "@azure-tools/cadl-azure-core";
 import "@azure-tools/cadl-ranch-expect";
 
 using Cadl.Rest;
 using Cadl.Http;
 using Azure.Core.Foundations;
 
-@doc("Illustrates round-trip polymorphic models.")
-@scenarioService("/polymorphic")
-namespace BasicPolymorphicModels;
+@serviceTitle("ComplexInheritanceClient")
```

Comment:
You can do `@scenarioService("/models/inheritance")` and just have doc, scenarioService, and namespace

Reply:
modified